wire-library/wire-schema-tests/src/main/java/com/squareup/wire/SchemaBuilder.kt

```diff
@@ -0,0 +1,82 @@
+package com.squareup.wire
```

Comment:
This is what users would use to test their SchemaHandlers.

Reply:
Nit: Copyright 2022